com.simplenote.Simplenote.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: com.simplenote.Simplenote
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: simplenote
+finish-args:
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --filesystem=xdg-config/Simplenote
```

Comment:
What does it needs this for?

Reply:
`share=ipc` without it, the X11 shared memory extension will not work, which is very bad for X11 performance.
`share=network` the notes are stored remotely and must be accessed through the network
`socket=cups` the ability to print your notes
`filesystem=xdg-config/Simplenote` where the default configuration is kept

Reply:
Are you sure it doesn't respect `$XDG_CONFIG_HOME`?

Reply:
I tested a build by removing `--filesystem=xdg-config/Simplenote` and it didn't seem to access my existing configuration. As it wanted me to log into the app like it was my first time. I then put that option back and rebuilt and launched and everything worked as expected.

Reply:
At first glance, it seems to properly use `/var/home/bpiotrowski/.var/app/com.simplenote.Simplenote/config` for me.